snafu/run_snafu.py

```diff
@@ -99,9 +108,23 @@ def main():
     # call py es bulk using a process generator to feed it ES documents
     if index_args.index_results:
         parallel_setting = strtobool(os.environ.get("parallel", "false"))
-        res_beg, res_end, res_suc, res_dup, res_fail, res_retry = streaming_bulk(
-            es, process_generator(index_args, parser), parallel_setting
-        )
+
+        if "archive" in index_args.tool:
+            if index_args.archive_file:
```

Comment:
@acalhounRH doesn't there need to be an if !create-archive as well with this? Otherwise if you are trying to create one w/ a specific name/dest it will try to read it instead. Or am I just horribly misreading the code :-D

Reply:
Using your own word, just horribly misreading the code :-D

If --create-archive
    create the archive file, if user provided file use that if not then use the default scheme
else: # user did not provide --create-archive 
   do your normal thing

Reply:
ack what i wasn't grasping was that you actually set a tool called archive for this. Makes sense now.